src/shared/components/ncTable/sections/CustomTable.vue

```diff
@@ -215,10 +219,10 @@ export default {
 	},
 
 	mounted() {
-		subscribe('tables:selected-rows:deselect', this.deselectAllRows)
+		subscribe('tables:selected-rows:deselect', tableId => { this.deselectAllRows(tableId) })
 	},
 	beforeDestroy() {
-		unsubscribe('tables:selected-rows:deselect', this.deselectAllRows)
+		unsubscribe('tables:selected-rows:deselect', tableId => { this.deselectAllRows(tableId) })
```

Comment:
Realizing that this is inefficient, and will result in too many events if there are multiple tables. Thinking subscribing on the parent instead and using event delegation instead